pandas/tests/io/formats/test_style.py

```diff
@@ -103,6 +103,12 @@ def test_render(self):
         s.render()
         # it worked?
 
+    def test_render_empty_df(self):
+        empty_df = DataFrame()
```

Comment:
You can add a link to the issue here if you want `https://github.com/pandas-dev/pandas/issues/15953`